jupyter/src/notebook/jupyter_notebook.ml

```diff
@@ -30,7 +30,7 @@ type display_id = string
 (** {2 Display} *)
 
 let display ?ctx ?display_id ?(metadata = `Assoc []) ?(base64 = false) mime data =
-  let data = if base64 then B64.encode data else data in
+  let data = if base64 then Base64.encode_exn data else data in
```

Comment:
And a lower bound on base64 (just so you know: this is incompatible with core_extended at the moment, due to the clashing with their Base64 implementation)